front/src/pods/create-session/text-field.component.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useField } from 'formik';
+import MuiTextField, { TextFieldProps } from '@material-ui/core/TextField';
+
+export const TextFieldComponent: React.FunctionComponent<TextFieldProps> = props => {
```

Comment:
The TextFieldComponent should be in common/components/formst

The validationSchema is the one that has to be on every pod

Reply:
LEt's create a separate issue for this, idea

```tsx
const useFormInfo = <T>(name : string, props :T) => {
  const [field, meta] = useField(props.name);
  const textFieldProps = Boolean(field) ? field : props;
  const hasError = Boolean(meta && meta.touched && meta.error);  
  return {value, error: meta.error, hasError}
}
```